.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "launch": {
+
+    "configurations": [],
+    "compounds": [],
+
+  },
+  "ruby.rubocop.executePath": "",
+  "ruby.format": "rubocop" // use rubocop for formatting
```

Comment:
If you want to keep the in your repo, it's better to use [editorconfig](https://editorconfig.org/). The vscode specific settings should go in the gitignore

Reply:
I have some trouble with rubocop, so I should remove those line anyways